egs/wsj/s5/utils/format_lm_sri.sh

```diff
@@ -85,30 +85,15 @@ mkdir -p $out_dir
 cp -r $lang_dir/* $out_dir || exit 1;
 
 lm_base=$(basename $lm '.gz')
-gunzip -c $lm | utils/find_arpa_oovs.pl $out_dir/words.txt \
-  > $out_dir/oovs_${lm_base}.txt || exit 1;
-
-# Removing all "illegal" combinations of <s> and </s>, which are supposed to
-# occur only at being/end of utt.  These can cause determinization failures
-# of CLG [ends up being epsilon cycles].
-gunzip -c $lm \
-  | egrep -v '<s> <s>|</s> <s>|</s> </s>' \
-  | gzip -c > $tmpdir/lm.gz || exit 1;
-
 awk '{print $1}' $out_dir/words.txt > $tmpdir/voc || exit 1;
 
 # Change the LM vocabulary to be the intersection of the current LM vocabulary
 # and the set of words in the pronunciation lexicon. This also renormalizes the
 # LM by recomputing the backoff weights, and remove those ngrams whose
 # probabilities are lower than the backed-off estimates.
-change-lm-vocab -vocab $tmpdir/voc -lm $tmpdir/lm.gz -write-lm $tmpdir/out_lm \
-  $srilm_opts || exit 1;
-
-arpa2fst $tmpdir/out_lm | fstprint \
-  | utils/eps2disambig.pl | utils/s2eps.pl \
-  | fstcompile --isymbols=$out_dir/words.txt --osymbols=$out_dir/words.txt \
-    --keep_isymbols=false --keep_osymbols=false \
-  | fstrmepsilon | fstarcsort --sort_type=ilabel > $out_dir/G.fst || exit 1;
+change-lm-vocab -vocab $tmpdir/voc -lm $lm -write-lm - $srilm_opts | \
+  arpa2fst --disambig-symbol=#0 \
+           --read-symbol-table=$out_dir/words.txt - $out_dir/G.fst || exit 1
 
```

Comment:
The new make_phone_graph.sh results in 0.1% degradation of the tri4a_reseg system:
%WER 22.8 | 1831 21395 | 79.8 13.7 6.4 2.7 22.8 62.2 | exp.test/tri4a_reseg/decode_eval2000_sw1_tg/score_16_1.0/eval2000.ctm.swbd.filt.sys

Baseline:
%WER 22.7 | 1831 21395 | 80.2 13.8 6.0 2.9 22.7 62.4 | exp/tri4a_reseg/decode_eval2000_sw1_tg/score_14_0.0/eval2000.ctm.swbd.filt.sys